src/services/collabReports.ts

```diff
@@ -135,11 +138,66 @@ export async function createOrUpdateReport(newReport, oldReport) {
   };
 }
 
-export function getReports() {
-  const stubbedSqlData = [];
-  const reports = stubbedSqlData;
-  return Promise.resolve({
-    count: reports.length,
-    rows: reports,
+export async function getReports(
+  {
+    sortBy = 'updatedAt',
+    sortDir = 'desc',
+    offset = 0,
+    limit = REPORTS_PER_PAGE,
+    status = REPORT_STATUSES.APPROVED,
+    userId = 0,
+    ...filters
+  }: {
+    sortBy?: string;
+    sortDir?: 'asc' | 'desc';
+    offset?: number;
+    limit?: number;
+    status?: keyof typeof REPORT_STATUSES | Array<keyof typeof REPORT_STATUSES>;
+    userId?: number;
+  } = {},
+) {
+  const { collabReports: scopes } = await filtersToScopes(filters, { userId });
+
+  return CollabReport.findAndCountAll({
+    where: {
+      [Op.and]: [
+        { calculatedStatus: status },
+        scopes,
+      ],
+    },
```

Comment:
Do we need to return all attributes here, or just the ones required for the table?